gremlin-go/README.md

```diff
@@ -269,6 +244,10 @@ TODO: Add Go specific changes to following paragraph:
 The Gremlin language allows users to write highly expressive graph traversals and has a broad list of functions that
 cover a wide body of features. Traversal in `Go` is very similar to other GLV, with the exception that all step functions are capitalized.
 
+Anonymous traversal `__` replaced with `T__` due to `Go` limitations.
+
+Constants and predicates should be prefixed with `gremlingo` like `gremlingo.Desc`.
```

Comment:
I think this needs a slight rewording, but I'm not sure how to word it exactly.

It's not just constants and predicates. Anything in the package when referenced needs the prefix. It's only after something is saved as a variable that it can be dropped

Reply:
Anything in the package when referenced needs the prefix `gremlingo` like `gremlingo.Desc`

Reply:
updated in https://github.com/lyndonbauto/tinkerpop/pull/88/commits/31a3b30caf421ac03e42c24c68c768e7fa7bb193